manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/api/user/SinkConnectorsAPI.java

```diff
@@ -0,0 +1,133 @@
+package com.redhat.service.smartevents.manager.v2.api.user;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotEmpty;
+import javax.ws.rs.BeanParam;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.eclipse.microprofile.openapi.annotations.Operation;
+import org.eclipse.microprofile.openapi.annotations.enums.SecuritySchemeType;
+import org.eclipse.microprofile.openapi.annotations.media.Content;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponse;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponses;
+import org.eclipse.microprofile.openapi.annotations.security.SecurityRequirement;
+import org.eclipse.microprofile.openapi.annotations.security.SecurityScheme;
+import org.eclipse.microprofile.openapi.annotations.security.SecuritySchemes;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import com.redhat.service.smartevents.infra.core.models.responses.ErrorsResponse;
+import com.redhat.service.smartevents.infra.v2.api.V2APIConstants;
+import com.redhat.service.smartevents.infra.v2.api.models.queries.QueryResourceInfo;
+import com.redhat.service.smartevents.manager.v2.api.user.models.requests.ConnectorRequest;
+import com.redhat.service.smartevents.manager.v2.api.user.models.responses.SinkConnectorListResponse;
+import com.redhat.service.smartevents.manager.v2.api.user.models.responses.SinkConnectorResponse;
+
+import io.quarkus.security.Authenticated;
+
+@Tag(name = "Sink Connectors", description = "The API that allow the user to retrieve, create or delete Sink Managed Connectors to be used with smart-events.")
```

Comment:
I don't have a specific preference honestly, I've removed the `Managed` atm just because it is shorter. I slightly prefer `Source Connectors` than `Source Managed Connectors`. If you think we should do the other way around, I change it with no problems :+1:

Reply:
"Source Connector" and "Sink Connector" are my preferences too :smiley: